misk/src/main/kotlin/misk/hibernate/DbTimestampedEntity.kt

```diff
@@ -0,0 +1,54 @@
+package misk.hibernate
+
+import java.time.Instant
+
+/**
+ * This complements [DbEntity] for entities that keep updated_at and created_at timestamps.
+ *
+ * Misk uses the application's clock time to set [created_at] and [updated_at]. We prefer the
+ * application's (ie. JVM's) clock over the database's clock because only the application's clock
+ * can be faked in tests.
```

Comment:
This isn't necessarily true... We could potentially do something here at the Vitess level...

Reply:
yeah, but that’s also pretty awkward to hook into FakeClock, right?